routes/voice-alexa.js

```diff
@@ -0,0 +1,498 @@
+        if (typeof this.event.request.intent.slots.Answer.value !== "undefined") {            
+            const sessionId = this.event.session.sessionId;        
+            const guessValue = this.event.request.intent.slots.Answer.value;
+            const currentQuestion = this.attributes['currentQuestion'];
+
+            checkGuess(sessionId, guessValue, currentQuestion, 
+                ((response, reprompt, state, card, increment) => {
+                if (card) {
+                    this.response.cardRenderer(card.title, card.body, card.image);
+                }
+                if (increment) {
+                    Object.assign(this.attributes, {
+                        'speechOutput': reprompt,
+                        'currentQuestion': this.attributes['currentQuestion'] + 1
+                    });
+                } 
+                this.handler.state = state;
+                this.response.speak(response).listen(reprompt);   
+                this.emit(':responseReady');
+            }));
+        } else {
+            this.handler.state = GAME_STATES.QUIZ;
+            this.emitWithState('Unhandled', true);
+        }
+    },   
+    'AMAZON.RepeatIntent': function () {
+        this.response.speak(this.attributes['speechOutput']).listen(this.attributes['speechOutput']);
+        this.emit(':responseReady');
+    },
+    'AMAZON.HelpIntent': function () {
+        this.handler.state = GAME_STATES.HELP;
+        this.emitWithState('helpTheUser', true);
+    },
+    'AMAZON.CancelIntent': function () {
+        const sessionId = this.event.session.sessionId;
+        
+        games.interrupt(sessionId).then(data => {
+            const response = responses.stop(true, {score: data.score, scoreMax: data.globalHighestScore, first: data.achievedHighestScoreFirst});
+            this.emit(':tell', response.speech);
+        });    
+    },
+    'AMAZON.StartOverIntent': function () {
+        const sessionId = this.event.session.sessionId;        
+        
+        this.handler.state = GAME_STATES.START;
+        games.interrupt(sessionId).then(data => {
+            this.emitWithState('StartGame', true);        
+        });
+    },
+    'AMAZON.StopIntent': function () {
+        this.handler.state = GAME_STATES.HELP;
+        this.response.speak(speech['ASK_CONTINUE']).listen(speech['ASK_CONTINUE']);
+        this.emit(':responseReady');
+    },
+    'Unhandled': function () {
+        this.response.speak(speech['QUIZ_UNHANDLED']).listen(speech['QUIZ_UNHANDLED']);        
+        this.emit(':responseReady');
+    }
+});
+
+const helpStateHandlers = Alexa.CreateStateHandler(GAME_STATES.HELP, {
+    'helpTheUser': function () {
+        const sessionId = this.event.session.sessionId;
+        
+        const helpBody = games.check(sessionId)
+        .then(gameIsInProgress => {
+            let helpResponse = responses.help(gameIsInProgress).ssml;
+            helpResponse = removeSpeakTags(helpResponse);    
```

Comment:
use the `.speech` property from the the response, avoids having to remove `<speak>` tags. Or ar the `<break>` tags supported by Alexa?

Reply:
Alexa automatically adds <speak> tags, so unfortunately this is necessary